checks/core/deployed_version.py

```diff
@@ -0,0 +1,37 @@
+"""
+The deployed `version` should be the latest tag of the specified `repo`.
+
+The deployed version and the latest tag are returned.
+"""
+import os
+
+import aiohttp
+
+from poucave.typings import CheckResult
+
+
+EXPOSED_PARAMETERS = ["server", "repo"]
+
+REQUESTS_TIMEOUT_SECONDS = int(os.getenv("REQUESTS_TIMEOUT_SECONDS", 5))
+
+
+async def run(server: str, repo: str) -> CheckResult:
+    timeout = aiohttp.ClientTimeout(total=REQUESTS_TIMEOUT_SECONDS)
+    async with aiohttp.ClientSession(timeout=timeout) as session:
+        # Server __version__ (see mozilla-services/Dockerflow)
+        async with session.get(server + "/__version__") as response:
+            version_info = await response.json()
+
+        # Latest Github tag.
+        async with session.get(
+            f"https://api.github.com/repos/{repo}/releases"
+        ) as response:
+            releases = await response.json()
+
+    deployed_version = version_info["version"]
+    latest_tag = releases[0]["tag_name"]
+
+    return (
+        deployed_version == latest_tag,
```

Comment:
This might be a nit, but can we also allow `v{deployed_version}`? That's how we tag e.g. kinto-http.js. https://github.com/Kinto/kinto-http.js/tree/v4.7.3

Reply:
Yeah, but I think we use the Github tag to produce the `version.json` so we should be safe

See https://github.com/mozilla/normandy/blob/master/.circleci/config.yml#L69